Dockerfile

```diff
@@ -27,10 +28,38 @@ ONBUILD ARG PGUSER=odoo
 ONBUILD ARG PGPASSWORD=odoopassword
 ONBUILD ARG PGHOST=db
 ONBUILD ARG PGDATABASE=prod
-ONBUILD ENV PGUSER="$PGUSER" \
+ONBUILD ARG ODOO_MAX_CRON_THREADS=1
+ONBUILD ARG ODOO_WORKERS=0
+ONBUILD ARG ODOO_LIMIT_TIME_REAL=120
+ONBUILD ARG ODOO_LIMIT_TIME_CPU=60
+ONBUILD ARG ODOO_LIMIT_MEMORY_HARD=805306368
+ONBUILD ARG ODOO_LIMIT_MEMORY_SOFT=671088640
+ONBUILD ARG ODOO_LIMIT_REQUEST=8192
+ONBUILD ARG ODOO_LIMIT_TIME_REAL_CRON=1
+ONBUILD ARG ODOO_DB_FILTER=
+ONBUILD ARG ODOO_LIST_DB=1
+ONBUILD ARG ODOO_LOG_LEVEL=info
```

Comment:
IMO because the variety of uses for this image, we shouldn't make too many assumptions about the deploy by default. 

Basically all I did here was expose some more configuration options, using specifically the ones [Odoo sets as default](https://github.com/odoo/odoo/blob/3ace806d399e9e3abc5ada84ba98d2de100388d3/odoo/tools/config.py#L277). 

I think the better configuration though would just be a part of the env files in scaffolds though:

```
odoo:
  env_file:
    - common.env
    - 32gbram.env
```